docs/index.bs

```diff
@@ -222,6 +222,13 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 
     A <dfn export id="dfn-service-worker-client" for="">service worker client</dfn> is an [=environment=].
 
+    A [=/service worker client=] has an associated <dfn export>discarded flag</dfn>. It is initially unset.
+
+   Each [=/service worker client=] has the following [=environment discarding steps=]:
+        1. Set |client|'s [=discarded flag=].
+
+    Note: Implementations can discard clients whose [=discarded flag=] is set.
```

Comment:
"Implementations can discard clients" sounds normative, but maybe it's defined somewhere else? Otherwise, add it to the definition of the flag. Eg:

```
A [=/service worker client=] has an associated <dfn export>discarded flag</dfn>. It is initially unset. User agents may discard clients whose [=discarded flag=] is set.
```

(it looks like we tend to say "user agents" rather than "implementations")

Reply:
Ah no, it was just supposed to be a hint for implementors. It just says "don't worry about this flag. you don't have to keep state around for a client with this flag set". I can just remove the non-normative note if it's confusing.

Reply:
Ohhh I get it now. Yeah that's fine then.

Reply:
@mattto, so what do you expect the implementations actually do when the discarded flag is set? That bit sounds like a normative to me. Let me know if closing/terminating of the clients is rather implicit so we don't have to specify. 

(And now we auto-bikeshed the .bs file in Nightly. So you can just upload .bs without .html next time. For this PR, don't worry about it.)

Reply:
@jungkees I think implementations would resolve any open clients.get(id) promise with undefined, then completely forget about that id. If clients.get(id) were called again, it would resolve with undefined immediately because the id wouldn't be found. So they don't need to keep state of the client around. The flag is just to make the spec algorithm work.

Sounds like I should just remove the note.

Reply:
Let's keep the note. I'll merge the PR. Maybe you can update the note later with what you explained if that'll make it clearer.